core/ir/x86/decode_table.c

```diff
@@ -1626,7 +1626,15 @@ const instr_info_t * const op_instr[] =
 
```

Comment:
I do a manual walk through the tables to ensure the entries are correct.  The decode_sizeof is a good sign but not necessarily exhaustive since api.ir often has just a single instance and many instructions have various choices for the operands.  It is rare for these small sets of added opcodes to need a new entry though as they often fill in a slot very similar to existing opcodes.

Reply:
I manually tested the multiple variants (added some logging to the `#define OPCODE` to confirm). Unfortunately due to the way the test is set up, looks like we cannot have more than one test for each opcode (complains about re-declaration of 'len_xsaves32' constant).

```
one reg + disp:
opcode 1437, len 4: 0f c7 69 37 xsaves %edx %eax -> 0x37(%rcx)
opcode 1438, len 5: 48 0f c7 69 37 xsaves64 %edx %eax -> 0x37(%rcx)
opcode 1439, len 4: 0f c7 59 37 xrstors 0x37(%rcx) %edx %eax
opcode 1440, len 5: 48 0f c7 59 37 xrstors64 0x37(%rcx) %edx %eax

one reg: 
opcode 1437, len 3: 0f c7 29 xsaves %edx %eax -> (%rcx)
opcode 1438, len 4: 48 0f c7 29 xsaves64 %edx %eax -> (%rcx)
opcode 1439, len 3: 0f c7 19 xrstors (%rcx) %edx %eax
opcode 1440, len 4: 48 0f c7 19 xrstors64 (%rcx) %edx %eax

two reg + disp:
opcode 1437, len 9: 42 0f c7 ac f0 23 01 00 00 xsaves %edx %eax -> 0x00000123(%rax,%r14,8)
opcode 1438, len 9: 4a 0f c7 ac f0 23 01 00 00 xsaves64 %edx %eax -> 0x00000123(%rax,%r14,8)
opcode 1439, len 9: 42 0f c7 9c f0 23 01 00 00 xrstors 0x00000123(%rax,%r14,8) %edx %eax
opcode 1440, len 9: 4a 0f c7 9c f0 23 01 00 00 xrstors64 0x00000123(%rax,%r14,8) %edx %eax

opcode 1437, len 8: 0f c7 ac f0 23 01 00 00 xsaves %edx %eax -> 0x00000123(%rax,%rsi,8)
opcode 1438, len 9: 48 0f c7 ac f0 23 01 00 00 xsaves64 %edx %eax -> 0x00000123(%rax,%rsi,8)
opcode 1439, len 8: 0f c7 9c f0 23 01 00 00 xrstors 0x00000123(%rax,%rsi,8) %edx %eax
opcode 1440, len 9: 48 0f c7 9c f0 23 01 00 00 xrstors64 0x00000123(%rax,%rsi,8) %edx %eax

two reg:
opcode 1437, len 5: 42 0f c7 2c 01 xsaves %edx %eax -> (%rcx,%r8)
opcode 1438, len 5: 4a 0f c7 2c 01 xsaves64 %edx %eax -> (%rcx,%r8)
opcode 1439, len 5: 42 0f c7 1c 01 xrstors (%rcx,%r8) %edx %eax
opcode 1440, len 5: 4a 0f c7 1c 01 xrstors64 (%rcx,%r8) %edx %eax
```